src/python/fuzzing/testcase_manager.py

```diff
@@ -471,12 +468,19 @@ def run_testcase_and_return_result_in_queue(crash_queue,
       # Don't upload uninteresting testcases (no crash) or if there is no log to
       # correlate it with (not upload_output).
       if upload_output:
-        upload_testcase(file_path)
+        log_time = _get_testcase_time(file_path)
+        upload_testcase(file_path, log_time)
 
     if upload_output:
       # Include full output for uploaded logs (crash output, merge output, etc).
       crash_result_full = CrashResult(return_code, crash_time, output)
-      upload_testcase_output(crash_result_full, file_path)
+
+      # To provide consistency between stats and logs, we use timestamp taken
+      # from stats.
+      log_time = _get_testcase_time(file_path)
```

Comment:
Can we save a _get_testcase_time call since it is called twice for upload_output branch

Reply:
Done!